### PR TITLE
Adding s1_orbits

### PR DESCRIPTION
**Pull request by jtherrmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by asjohnston-asf** [comment]
I'm willing to be a maintainer.

**Comment by jtherrmann** [comment]
I'm willing to be a maintainer.

**Comment by jacquelynsmale** [comment]
I'm willing to be a maintainer

**Comment by jhkennedy** [comment]
I am willing to be a maintainer.
image

**Comment by forrestfwilliams** [comment]
I'm willing to be a maintainer of this package

**Comment by jtherrmann** [comment]
@conda-forge/help-python, ready for review!

**Comment by jhkennedy** [comment]
@conda-forge-admin, please lint

**Comment by jhkennedy** [comment]
I'm not sure why the linter isn't catching @AndrewPlayer3's [comment](https://github.com/conda-forge/staged-recipes/pull/27200#issuecomment-2274125094); it's possibly case-sensitive. @jtherrmann want to match the capitalization?

File: recipes/s1_orbits/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+about:
+  home: https://github.com/ASFHyP3/sentinel1-orbits-py
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: A Python utility for downloading Sentinel-1 Orbit files from the Registry of Open Data on AWS
+
+extra:
+  recipe-maintainers:
+    - andrewplayer3
```
Comment (jtherrmann) [inline recipes/s1_orbits/meta.yaml:41]:
```suggestion
    - AndrewPlayer3
```

**Comment by jtherrmann** [comment]
@conda-forge-admin, please lint

File: recipes/s1_orbits/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e622a97b2d666c59c07b16fa89c2989f347999bf2a66e5fca4368ce47155cd7b
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/s1_orbits/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/s1_orbits/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - setuptools-scm >=6.2
+    - python-build
+  run:
+    - python >=3.9
+    - requests
+
+test:
+  imports:
+    - s1_orbits
+
```
Comment (ocefpaf) [inline recipes/s1_orbits/meta.yaml:31]:
```suggestion
  commands:
    - pip check
  requires:
    - pip

```

**Review by ocefpaf — COMMENTED** [review]
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.

**Comment by jtherrmann** [comment]
@ocefpaf Done.

**Pull request merged**